pkg/api/types.go

```diff
@@ -1662,6 +1665,27 @@ type SecretList struct {
 	Items []Secret `json:"items"`
 }
 
+// SecurityContextSpec holds security configuration that will be applied to the pod's containers
+type SecurityContextSpec struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
```

Comment:
Also, pkg/api/types.go objects should never be serialized, so the versioned APIs would also need to be updated.